docs/guidelines.md

```diff
@@ -13,8 +13,14 @@ Plugins with required `inputs` cannot be installed via the Netlify UI. Wherever
 Every plugin in the Netlify plugins directory includes a link to the plugin README. This file should include:
 
 - A description of what the plugin does and why that might be useful.
+- UI-based installation instructions. 
+  > **Note:** You can include UI-based installation instructions in a separate, follow-up pull request. The PR will be merged when your plugin is approved for inclusion in the plugins directory.
```

Comment:
This is per @erezrokah's suggestion. Let me know if this wording (or any wording in this PR) needs to be tweaked.

Reply:
This is looking good to me :+1: